app/views/admin/schedules/_day_tab.html.haml

```diff
@@ -1,18 +1,24 @@
+- cell_duration = EventType::LENGTH_STEP
+- compact_grid = EventType::LENGTH_STEP < 15
+- cells_per_hour = 60 / cell_duration
+/ use smaller cell heights for more compact grids
+- cell_height = compact_grid ? 32 : 58
 - date_event_schedules = @event_schedules.select{ |e| e.start_time.to_date.eql? date }
 .row
   - @rooms.each do |room|
     .col-md-2.col-xs-6
       .room-name
         - room_date_event_schedules = date_event_schedules.select{ |e| e.room == room }
         = room.name
-      - (9*4..18*4).each do |slot|
-        - hour = slot / 4
-        - minutes = (15 * (slot % 4) == 0) ? '00' : 15 * (slot % 4)
+      - (SCHEDULE_START_HOUR * cells_per_hour..SCHEDULE_END_HOUR * cells_per_hour).each do |slot|
+        - hour = slot / cells_per_hour
+        - minutes = (cell_duration * (slot % cells_per_hour)).to_s.rjust(2, '0')
         - time = "#{hour}:#{minutes}"
         .schedule-room-slot{ id: "schedule-room-#{room.guid}-#{hour}-#{minutes}", |
                              room_id: room.id, |
                              hour: time, |
-                             date: date}
+                             date: date, |
+                             style: "height: #{cell_height}px"}
```

Comment:
But then you need to go to the definition of all the variables to understand what `cell_height` is, while with that it would be just in one place. But yes, what others thing? @differentreality @ChrisBr ^